tests/unit/lms/services/mailchimp_test.py

```diff
@@ -14,16 +14,75 @@
 
 
 class TestSend:
+    @pytest.mark.parametrize(
+        "template,template_vars,expected_subject",
+        [
+            (
+                "lms:templates/email/instructor_email_digest/",
+                {
+                    "total_annotations": 2,
+                    "annotators": [sentinel.annotator_1, sentinel.annotator_2],
+                    "courses": [sentinel.course_1, sentinel.course_2],
+                },
+                "Hypothesis: 2 annotations from 2 students in 2 courses",
+            ),
+            (
+                "lms:templates/email/instructor_email_digest/",
+                {
+                    "total_annotations": 2,
+                    "annotators": [sentinel.annotator_1, sentinel.annotator_2],
+                    "courses": [sentinel.course_1],
+                },
+                "Hypothesis: 2 annotations from 2 students in 1 course",
+            ),
+            (
+                "lms:templates/email/instructor_email_digest/",
+                {
+                    "total_annotations": 2,
+                    "annotators": [sentinel.annotator_1],
+                    "courses": [sentinel.course_1, sentinel.course_2],
+                },
+                "Hypothesis: 2 annotations from 1 student in 2 courses",
+            ),
+            (
+                "lms:templates/email/instructor_email_digest/",
+                {
+                    "total_annotations": 2,
+                    "annotators": [sentinel.annotator_1],
+                    "courses": [sentinel.course_1],
+                },
+                "Hypothesis: 2 annotations from 1 student in 1 course",
+            ),
+            (
+                "lms:templates/email/instructor_email_digest/",
+                {
+                    "total_annotations": 1,
+                    "annotators": [sentinel.annotator_1],
+                    "courses": [sentinel.course_1],
+                },
+                "Hypothesis: one of your students made an annotation",
+            ),
+        ],
+    )
```

Comment:
It's a bit odd that the tests for the digest email template are here in `MailchimpService`'s tests rather than in `DigestService`'s tests, but `MailchimpService` is what actually renders the template. If we ever add more templates for more emails we can add them to this parametrize